tests/chainer_tests/test_function.py

```diff
@@ -362,5 +381,18 @@ def test_debug_backward_gpu(self):
             return_value=(cuda.to_gpu(self.nan),))
         self.check_debug_backward(cuda.to_gpu(self.one))
 
+    def test_debug_backward_none_cpu(self):
+        self.f.forward_cpu = mock.MagicMock(return_value=(self.one,))
+        self.f.backward_cpu = mock.MagicMock(return_value=(None,))
+        self.check_debug_backward(self.one, valid=True)
+
```

Comment:
Can we make make `test_debug_backward_cpu` and `test_debug_backward_none_cpu` parameterized with respect to `return_value` of backward and `valid`? (We may need to separate forward tests (e.g. `test_debug_forward_cpu`) and backward tests (e.g. `test_debug_backward_cpu`)).